app/namespace.go

```diff
@@ -978,6 +1013,168 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 						},
 					},
 				},
+				{
+					Name:    "export-sinks",
+					Aliases: []string{"es"},
+					Usage:   "Manage namespace's export sinks",
+					Subcommands: []*cli.Command{
+						{
+							Name:    "create",
+							Aliases: []string{"c"},
+							Flags: []cli.Flag{
+								NamespaceFlag,
+								sinkNameFlag,
+								sinkEnabledFlag,
+								sinkTypeFlag,
+								sinkAssumedRoleFlag,
+								sinkDestinationArnFlag,
+								&cli.StringFlag{
```

Comment:
we should probably indicate the region must match that of the namespace here? or maybe not include the region and let the server derive it from the namespace?